spec/ameba/rule/style/percent_literal_delimiters_spec.cr

```diff
@@ -26,6 +26,38 @@ module Ameba::Rule::Style
         CRYSTAL
     end
 
+    it "corrects incorrect percent literal delimiters" do
+      source = expect_issue subject, <<-CRYSTAL
+        %[one two three]
```

Comment:
Should add a test for `%[)]` or other similar cases of these containing other delimiters